pkgs/tools/networking/openconnect/common.nix

```diff
@@ -15,6 +15,8 @@
 , zlib
 , vpnc-scripts
 , PCSC
+, useDefaultExternalBrowser ? lib.versionAtLeast version "9.00"
```

Comment:
Is this worth complicating stuff? AFAICS we only package versions that are >= 9.00

Reply:
Users could overlay a version less than 9.00 right? That said, not sure it's a huge deal having xdg-utils as an unused dependency in that scenario.

Reply:
Yes, and users can also override buildInputs in the overlay.

Reply:
@prusnak made and tested the change.